cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,284 @@
+        description:
+            - Determines wether the backend is to be created/modified \
+            or deleted
+    probe:
+        required: false
+        default: none
+        choices: ['none', 'http', 'icmp' , 'oco']
+        description:
+            - Determines the type of probe to use for this backend
+    weight:
+        required: false
+        default: 8
+        description:
+            - Determines the weight for this backend
+    endpoint:
+        required: true
+        description:
+            - The endpoint to use ( for instance ovh-eu)
+    application_key:
+        required: true
+        description:
+            - The applicationKey to use
+    application_secret:
+        required: true
+        description:
+            - The application secret to use
+    consumer_key:
+        required: true
+        description:
+            - The consumer key to use
+    timeout:
+        required: false
+        type: "int"
+        default: 120
+        description:
+            - The timeout in seconds used to wait for a task to be \
+            completed. Default is 120 seconds.
+
+'''
+
+EXAMPLES = '''
+# Adds or modify a backend to a loadbalancing
+- ovh_ip_loadbalancing name=ip-1.1.1.1 ip=212.1.1.1 state=present \
+probe=none weight=8 \
+endpoint=ovh-eu application_key=yourkey \
+application_secret=yoursecret consumer_key=yourconsumerkey
+
+# Removes a backend from a loadbalancing
+- ovh_ip_loadbalancing name=ip-1.1.1.1 ip=212.1.1.1 state=absent \
+endpoint=ovh-eu application_key=yourkey \
+application_secret=yoursecret consumer_key=yourconsumerkey
+'''
+
+
+def getOvhClient(ansibleModule):
+    endpoint = ansibleModule.params.get('endpoint')
+    application_key = ansibleModule.params.get('application_key')
+    application_secret = ansibleModule.params.get('application_secret')
+    consumer_key = ansibleModule.params.get('consumer_key')
+
+    return ovh.Client(
+        endpoint=endpoint,
+        application_key=application_key,
+        application_secret=application_secret,
+        consumer_key=consumer_key
+    )
+
+
+def waitForNoTask(client, name, timeout):
+    currentTimeout = timeout
+    while len(client.get('/ip/loadBalancing/{}/task'.format(name))) > 0:
+        time.sleep(1)  # Delay for 1 sec
+        currentTimeout -= 1
+        if currentTimeout < 0:
+            return False
+    return True
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(required=True),
+            backend=dict(required=True),
+            weight=dict(default=8, type='int'),
+            probe=dict(default='none',
+                       choices=['none', 'http', 'icmp', 'oco']),
+            state=dict(default='present', choices=['present', 'absent']),
+            endpoint=dict(required=True),
+            application_key=dict(required=True),
+            application_secret=dict(required=True),
+            consumer_key=dict(required=True),
```

Comment:
Consumer_key and application secret need to have no_log=True to be set, cf https://docs.ansible.com/ansible/developing_modules.html#module-checklist , sorry, I forgot about this one